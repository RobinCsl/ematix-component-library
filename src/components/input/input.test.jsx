import React from "react";
import renderer from "react-test-renderer";

import Input from "./index";

it("renders correctly", () => {
  const tree = renderer
    .create(<div><Input /></div>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
