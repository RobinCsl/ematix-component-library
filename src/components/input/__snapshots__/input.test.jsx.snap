// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div>
    <input
      type="text"
      value="test"
    />
  </div>
</div>
`;
